CHANGELOG.md

```diff
@@ -15,7 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   type. `tarantool.Datetime` may be encoded to Tarantool datetime
   objects.
 
-  You can create `tarantool.Datetime` objects either from msgpack
+  You can create `tarantool.Datetime` objects either from MsgPack
```

Comment:
The fullname is MessagePack: https://msgpack.org/

```suggestion
  You can create `tarantool.Datetime` objects either from MessagePack
```

Reply:
There is also an inconsistency here:

tarantool/msgpack_ext/uuid.py:The UUID MessagePack representation looks like this:
tarantool/msgpack_ext/types/interval.py:The interval MessagePack representation looks like this:
tarantool/msgpack_ext/types/datetime.py:The datetime MessagePack representation looks like this:
tarantool/msgpack_ext/types/datetime.py:MessagePack data contains:
tarantool/msgpack_ext/decimal.py:The decimal MessagePack representation looks like this:

Reply:
Remade to MessagePack where seemed relevant